testsuite/smoke-test/src/full_nodes.rs

```diff
@@ -2,8 +2,81 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::{smoke_test_environment::SmokeTestEnvironment, test_utils::compare_balances};
+use cli::client_proxy::{ClientProxy, IndexAndSequence};
+use diem_client::AccountAddress;
 use diem_types::account_config::{testnet_dd_account_address, treasury_compliance_account_address};
 
+// TODO: All of this convenience code below should be put in the client proxy directly, it's
+// very hard for users to know what the order of a bunch of random inputs should be
```

Comment:
TBH we shouldn't be relying on the little test cli in these tests. I'm writing a suite of tests right now only using the SDK and I'm finding lots of areas for improvement.

Reply:
I agree, we shouldn't be using this CLI at all in here.